provision.yaml

```diff
@@ -0,0 +1,19 @@
+---
+default:
+  provisioner: docker_exp
+  images: ['waffleimage/centos7']
+waffle_deb:
+  provisioner: docker_exp
+  images: ['waffleimage/debian8', 'waffleimage/debian9', 'waffleimage/ubuntu14.04', 'waffleimage/ubuntu16.04', 'waffleimage/ubuntu18.04']
+waffle_el6:
```

Comment:
seeing as the tests are quick to run, we should combine the el6 and el7 groups. And just run one group, not 2 for EL

Reply:
@tphoney Looking at the Travis config, it looks like it is distinguishing between the EL5 and EL6 platforms? If those can be safely squashed I'm happy to do this, digging in now.